### ISSUE TITLE
Interacting with CC Library asset can result in error dialog, file damaged

### ISSUE THREAD

**Issue by chadrolfs** [adobe-photoshop/spaces-design#2353]
I've seen this twice this morning.  I'll work on better repro steps but it has popped up using the same CC Lib asset that's a vector shape.

Conditions: multiple artboards (some with auto-nesting locked).  Error occurred after dragging out CC Lib onto canvas (not onto AB), then moving or transforming it.

<img width="425" alt="screen shot 2015-09-08 at 7 23 55 am" src="https://cloud.githubusercontent.com/assets/8442631/9737534/ee91e656-55fb-11e5-96e5-d6e30fe30653.png">

**Comment by chadrolfs** [comment]
here are the actions leading up to it:

<img width="1042" alt="screen shot 2015-09-08 at 7 24 34 am" src="https://cloud.githubusercontent.com/assets/8442631/9737546/ffd2da92-55fb-11e5-9e36-f45acee1804e.png">

**Comment by chadrolfs** [comment]
@shaoshing I'm gonna see if I can still repro this in latest builds.

**Comment by shaoshing** [comment]
Thank you Chad. The issue looks scary, although I have never encounter it, and your console seems normal except the drag events. I will give it a look.

**Comment by chadrolfs** [comment]
just repro'd on Windows
![ice_screenshot_20150914-161446](https://cloud.githubusercontent.com/assets/8442631/9864173/9b7f55d4-5afb-11e5-8005-efb0ce59de2b.png)

**Comment by chadrolfs** [comment]
Think I have a solid repro case.
Open the file I'm going to send you
Select the `Weather Auto-Nest Locked` AB
Load the Library I have shared with you `Test - Ok to accidentally nuke`
Drag TinaProject - Drawing 5 onto canvas, hit Enter to place
click and drag top right corner to scale
Result: error

https://www.dropbox.com/s/wx59laf4ctv9pmp/CCLib-FileDamageError.mov?dl=0

**Comment by shaoshing** [comment]
I can reproduce this in classic PS too (dev 606, https://www.dropbox.com/s/ymj1yejopdp8vuj/file-damaged.mov?dl=0), so I was thinking the DS side libraries might created the graphic asset incorrectly. Then I tried to create the same asset in classical PS, as a comparison, but failed due to another error "Could not create a library graphic because of a program error." (https://www.dropbox.com/s/tdis6voielifyem/add-graphic.mov?dl=0), which is weird because we used to be able to create asset for it. Further more, the The official release (r.168) passes the above two tests. I think it is very like a bug introduced from recent builds.
